spec/datetime.html

```diff
@@ -816,6 +805,25 @@ <h1>ConstrainDateTime ( _year_, _month_, _day_, _hour_, _minute_, _second_, _mil
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-temporal-balancedatetime" aoid="BalanceDateTime">
+      <h1>BalanceDateTime</h1>
```

Comment:
It'll be nice to add a `<p>` talking a little about `BalanceDateTime`. No hurries though, I could add one later while scrubbing the spec for Stage 3.

Reply:
What about arguments though? I'd really like if you added arguments to this line.

Reply:
Oops! I've added the arguments, but I skipped the `<p>` for now because none of the other operations have one and I'm trying to land this quickly :sweat_smile: